### ISSUE TITLE
TravisCI show regression test failing, but TravisCI passed

### ISSUE THREAD

**Issue by mdhorn** [clearlinux/clr-installer#441]
**Describe the bug**
There are two tests failing coverage checks, but TravisCI and coverage still pass.

**To Reproduce**
1. Submit a branch to TravisCI
2. Review the "passing" results in log details.

**Expected behavior**
If a test FAIL, the regress should fail.

**Screenshots**
```
2019/06/11 22:18:32 [INF] Wired networking managed by NetworkManager.service
--- FAIL: TestApply (0.03s)
    network_test.go:231: open /tmp/clr-installer-utest842103530/etc/NetworkManager/system-connections/Wired-docker0.nmconnection: no such file or directory
        
        Error Trace:
        errors.Wrap()
             errors/errors.go:91
        network.(*Interface).ApplyNetworkManager()
             network/network.go:614
        
    network_test.go:236: open /tmp/clr-installer-utest842103530/etc/NetworkManager/system-connections/Wired-docker0.nmconnection: no such file or directory
        
        Error Trace:
        errors.Wrap()
             errors/errors.go:91
        network.(*Interface).ApplyNetworkManager()
             network/network.go:614
        
2019/06/11 22:18:32 [INF] [Previous line repeated 1 time]
2019/06/11 22:18:32 [INF] Copied image Network configuration data to /tmp/clr-installer-utest534456129/etc/NetworkManager/system-connections
FAIL
coverage: 55.8% of statements
FAIL	github.com/clearlinux/clr-installer/network	2.493s
?   	github.com/clearlinux/clr-installer/progress	[no test files]
2019/06/11 22:18:32 [INF] Writing partition table to: loop0
2019/06/11 22:18:32 [INF] Updating partition table for: loop0
2019/06/11 22:18:32 [INF] Adjusting filesystem configurations
--- FAIL: TestWritePartition (4.24s)
    storage_test.go:513: Image file is :/tmp/test-image-572278349
    storage_test.go:590: Could not MakeFs partition (loop0p1): exit status 1
        
        Error Trace:
        errors.Wrap()
             errors/errors.go:91
        storage.makeFs()
             storage/ops.go:79
2019/06/11 22:18:36 [WRN] Unknown file system type unknown, defaulting to 11 character label
FAIL
coverage: 42.1% of statements
FAIL	github.com/clearlinux/clr-installer/storage	4.316s

```

**Comment by karthikprabhuvinod** [comment]
@mdhorn Looks you spotted this a while back.. I think can be closed due to this recent fix.

https://github.com/clearlinux/clr-installer/pull/680

**Comment by mdhorn** [comment]
Yep, Fixed by #680

**Comment by mdhorn** [comment]
Released in [2.4.3](https://github.com/clearlinux/clr-installer/releases/tag/2.4.3) in Clear Linux OS 32750 or higher.
